### PR TITLE
 Readme for health package

### PR DESCRIPTION
**Pull request by Evgeniy-L**

### CODE REVIEW

File: health/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Health Package
+
+Package contains tools that allow to add REST endpoint for handling `health` and `readiness` requests.
+Package also provides implementation for most typical checks that may be done to figure our if service is healthy/ready.
+
+## REST Handler
```
Comment (ZachEddy) [inline health/README.md]: It's not really a REST API, so much as just a plain HTTP Endpoint that serves one specific purpose. IMO, I think you can just say _HTTP Handler_ or _Probe Endpoints_. If you're looking for synonyms, maybe check out the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/).

File: health/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	)
+```
+
+In a sample above `readinessCheck` and `livenessCheck` are functions that perform a single check.
+It is possible to add multiple checks independently and handler will call them in sequence.
+
+## Checks
+
+Packange provides several predefined checks:
+ - `DNSProbeCheck(host string, timeout time.Duration)` returns a Check that determines whether a service with specified dns name is reachable or not using net.Resolver's LookupHost method.
```
Comment (ZachEddy) [inline health/README.md]: "dns" should be uppercase.

File: health/README.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+ - `HTTPGetCheck(url string, timeout time.Duration)` returns a Check that performs an HTTP GET request to the specified URL. It fails if timeout was exceeded or non-200-OK status code returned.
```
Comment (ZachEddy) [inline health/README.md]: Might help to change Check to `Check` to denote that it's a `struct` inside the package.

File: health/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Health Package
```
[the remark this answers is not in the document]
Comment (Evgeniy-L) [reply health/README.md]: I picked this name as it is a most common pattern. We already have `RequestID Package`, `Authorization/Authentication Package`, `Gateway Package`, etc. Lets stick to this pattern.

**Pull request merged**